packages/react-frontend/src/routes/FooterFiles/termscond.jsx

```diff
@@ -0,0 +1,6152 @@
+import React from "react"
+import { Link } from "react-router-dom"
```

Comment:
is there a terms and conditions css file yet?

Reply:
There is no css file. It is all implemented in termscond.jsx instead.

Reply:
is there a benefit to doing the css in a jsx file? just curious because i've only ever used css for styling